airmozilla/main/urls.py

```diff
@@ -75,6 +75,9 @@
     url(r'^(?P<slug>[-\w]+)/(?P<id>\d+)/difference/$',
         views.EventRevisionView.as_view(difference=True),
         name='event_difference'),
+    url(r'^eventstatus/(?P<id>\d+)/$',
+        views.event_status,
+        name='event_status'),
```

Comment:
Let's use the slug instead. So make the regex 
```python
r'^(?P<slug>[-\w]+)/status/$'
```
instead. That way, you never have to define the URL somewhere in the HTML for the JS to pick up. You can always find the URL by simply taking `document.location.pathname + 'status/'`, right?